third_party/BUILD

```diff
@@ -356,6 +356,20 @@ java_import(
 )
 
 java_import(
+    name = "scala-xml",
+    jars = [
+        "scala-xml/scala-xml-2.11.0-M4.jar",
```

Comment:
I think it is better to download rather than add the jar to the git repo. This can be done by adding to the WORKSPACE.